projects/packages/forms/src/contact-form/class-contact-form.php

```diff
@@ -461,6 +496,17 @@ public static function parse( $attributes, $content ) {
 		return apply_filters( 'jetpack_contact_form_html', $r );
 	}
 
+	/**
+	 * Function that adds the enctype attribute to the form element attributes.
+	 * This is useful for addin the myultipart/form-data to the form HTML element.
```

Comment:
Typo `myultipart`

Reply:
This should be fixed in https://github.com/Automattic/jetpack/pull/41693/commits/37710e7857b27f9bff74ae9c996c9d5fcefc1405